README.md

```diff
@@ -224,7 +224,30 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 | v0.29.0                | v1.19 to v1.21       |
 | v0.28.0                | v1.19 to v1.21       |
 | v0.27.0                | v1.19 to v1.21       |
-| v0.26.0                | v1.18 to v1.20       |
+
+
+### OpenTelemetry Operator vs. Cert Manager
+Since Cert-Manager deprecated *cert-manager.io/v1alpha2, cert-manager.io/v1alpha3, cert-manager.io/v1beta1, acme.cert-manager.io/v1alpha2, acme.cert-manager.io/v1alpha3, acme.cert-manager.io/v1beta1* APIs on version 1.4.0, we decided to create a compability Matrix between OpenTelemetry Operator and Cert-manager.
+
+| OpenTelemetry Operator | Cert-Manager         |
```

Comment:
How about adding a new column to the table from the previous section?

Reply:
Do you mean regarding the API versions?

Reply:
I mean the one with the Kubernetes versions. The title from the previous section could be changed from "OpenTelemetry Operator vs. Kubernetes" to "OpenTelemetry Operator vs. Kubernetes vs. cert-manager"

Reply:
Deal.